packages/react/src/lib/shouldHandleOnKeys.ts

```diff
@@ -3,18 +3,26 @@ import * as keyboardKey from 'keyboard-key'
 import * as _ from 'lodash'
 import * as React from 'react'
 
+const shouldHandleModifier = (modifierValue: boolean, combinationValue?: boolean) => {
```

Comment:
nit: `isKeyModifiersMatch` will provide better idea on what this method is designed to check